uom-plugin/tests/Tests.hs

```diff
@@ -117,7 +118,10 @@ baf qa qb = baz qa qb undefined
 
 -- Inferring this type used to lead to unit equations with occur-check
 -- failures, because it involves things like Pack (Unpack u) ~ u
-z q = convert q
+z :: forall a (u :: Unit) (v :: Unit).  (Fractional a, Convertible u v)
+  => Quantity a u
+  -> Quantity a v
```

Comment:
This type signature needs to be omitted, because the point of the test is to check that it can be inferred. 😄